lib/crypto/eosdart/src/models/account.g.dart

```diff
@@ -8,55 +8,30 @@ part of 'account.dart';
 
 Account _$AccountFromJson(Map<String, dynamic> json) {
   return Account(json['account_name'] as String?, json['head_block_num'] as int?)
-    ..headBlockTime = json['head_block_time'] == null
-        ? null
-        : DateTime.parse(json['head_block_time'] as String)
+    ..headBlockTime = json['head_block_time'] == null ? null : DateTime.parse(json['head_block_time'] as String)
     ..privileged = json['privileged'] as bool?
-    ..lastCodeUpdate = json['last_code_update'] == null
-        ? null
-        : DateTime.parse(json['last_code_update'] as String)
-    ..created = json['created'] == null
-        ? null
-        : DateTime.parse(json['created'] as String)
-    ..coreLiquidBalance = json['core_liquid_balance'] == null
-        ? null
-        : Holding.fromJson(json['core_liquid_balance'] as String)
-    ..ramQuota = json['ram_quota'] == null
-        ? null
-        : ConversionHelper.getIntFromJson(json['ram_quota'])
-    ..netWeight = json['net_weight'] == null
-        ? null
-        : ConversionHelper.getIntFromJson(json['net_weight'])
-    ..cpuWeight = json['cpu_weight'] == null
-        ? null
-        : ConversionHelper.getIntFromJson(json['cpu_weight'])
-    ..netLimit = json['net_limit'] == null
-        ? null
-        : Limit.fromJson(json['net_limit'] as Map<String, dynamic>)
-    ..cpuLimit = json['cpu_limit'] == null
-        ? null
-        : Limit.fromJson(json['cpu_limit'] as Map<String, dynamic>)
-    ..ramUsage = json['ram_usage'] == null
-        ? null
-        : ConversionHelper.getIntFromJson(json['ram_usage'])
+    ..lastCodeUpdate = json['last_code_update'] == null ? null : DateTime.parse(json['last_code_update'] as String)
+    ..created = json['created'] == null ? null : DateTime.parse(json['created'] as String)
+    ..coreLiquidBalance =
+        json['core_liquid_balance'] == null ? null : Holding.fromJson(json['core_liquid_balance'] as String)
+    ..ramQuota = json['ram_quota'] == null ? null : ConversionHelper.getIntFromJson(json['ram_quota'])
+    ..netWeight = json['net_weight'] == null ? null : ConversionHelper.getIntFromJson(json['net_weight'])
+    ..cpuWeight = json['cpu_weight'] == null ? null : ConversionHelper.getIntFromJson(json['cpu_weight'])
+    ..netLimit = json['net_limit'] == null ? null : Limit.fromJson(json['net_limit'] as Map<String, dynamic>)
+    ..cpuLimit = json['cpu_limit'] == null ? null : Limit.fromJson(json['cpu_limit'] as Map<String, dynamic>)
+    ..ramUsage = json['ram_usage'] == null ? null : ConversionHelper.getIntFromJson(json['ram_usage'])
     ..totalResources = json['total_resources'] == null
         ? null
-        : TotalResources.fromJson(
-            json['total_resources'] as Map<String, dynamic>)
+        : TotalResources.fromJson(json['total_resources'] as Map<String, dynamic>)
     ..permissions = (json['permissions'] as List?)
```

Comment:
another "as List" cast

Reply:
I think the problem is not using as list the problem is the type of the list defined in by the field List<SomeType>.
In fact, These models asume for almost all their filed are nullable when the docs says they have a default value.

Reply:
If the field is nullable the docs explicity mention it.
![image](https://user-images.githubusercontent.com/42857405/150047801-32929ee8-47b3-4b7c-bd2a-da4ff9988e4d.png)

Reply:
Great research. 

Sometimes reading the manual helps 😄